### PR TITLE
Feature/passive event listeners

### PR DESCRIPTION
**Pull request by timbielawski**
Allow for passive config on the event listeners

### CODE REVIEW

File: src/InfiniteScroll.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default class InfiniteScroll extends Component {
     threshold: PropTypes.number,
     useCapture: PropTypes.bool,
     useWindow: PropTypes.bool,
+    passive: PropTypes.bool,
```
Comment (joshuaobrien) [inline src/InfiniteScroll.js]: Will a consumer ever actually use this prop? I can't think of a situation where a consumer would ever want to not have a passive listener, but I may be missing something.

File: src/InfiniteScroll.js [hunk 2]
```diff
@@ -30,16 +31,20 @@ export default class InfiniteScroll extends Component {
     useCapture: false,
     loader: null,
     getScrollParent: null,
+    passive: false,
```
Comment (joshuaobrien) [inline src/InfiniteScroll.js]: As above

File: src/InfiniteScroll.js (hunk 1 above)
Comment (timbielawski) [reply src/InfiniteScroll.js]: I agree this has been removed

File: src/InfiniteScroll.js (hunk 2 above)
Comment (timbielawski) [reply src/InfiniteScroll.js]: I agree this has been removed thanks

File: src/InfiniteScroll.js [hunk 3]
```diff
@@ -68,6 +66,7 @@ export default class InfiniteScroll extends Component {
 
     try {
       document.addEventListener('test', null, testOptions);
+      document.removeEventListener('test', null, testOptions);
```
Comment (timbielawski) [inline src/InfiniteScroll.js]: event listener removed, thanks for feedback

**Comment by danbovey** [comment]
Released in `1.2.4`!

**Pull request merged**